agent/util-scripts/pbench-tool-meister-start

```diff
@@ -301,34 +300,25 @@ def main(_prog, cli_params):
             "The required 'pbench_install_dir' environment variable appears to be missing"
         )
         return ReturnCode.BADAGENTCONFIG
-    except Exception as exc:
+    try:
+        tm_start_path = Path(inst_dir).resolve(strict=True)
```

Comment:
This `try` block used to be in the `else` clause of the previous `try` block.  While I'm generally happy to see these `else` clauses removed, I'm suspicious that it was done by accident, here (i.e., there is still one at line 316).  Did you mean to do this?

Reply:
Yes this was intentional.  I removed the `else:` for `inst_dir`, but should have also removed the `else:` for `tm_start_path`.  I believe this is further refactored in PR #2181.